sklearn/compose/tests/test_column_transformer.py

```diff
@@ -297,6 +305,14 @@ def test_column_transformer_empty_columns(pandas, column):
     assert len(ct.transformers_) == 2  # including remainder
     assert isinstance(ct.transformers_[0][1], Trans)
 
+    # trans2 is not passed anything because column does not select any
```

Comment:
no, it's not even fitted. I think, rather, you should be using TransRaise in some of the above `ct` definitions, rather than introducing a new case.